pcdsdevices/attenuator.py

```diff
@@ -351,3 +352,44 @@ def set_combined_attenuation(attenuation, *attenuators):
         else:
             attenuators[i].actuate_value()
 '''
+
+
+class FEESolidFilter(Device, BaseInterface):
```

Comment:
```suggestion
class FEESolidAttenuator(Device, BaseInterface):
```
Because it feels wrong to refer to it four times in the docstring as an attenuator but then call it a filter

Reply:
Agreed